### ISSUE TITLE
Display changes in a log

### ISSUE THREAD

**Issue by JayFoxRox** [andir/ipv6.watch#58]
It would be cool if the past changes were listed, so it would be more obvious who adopted IPv6 lately.

**Comment by andir** [comment]
Agreed. I have been thinking about that for a while now. So far I didn't want to introduce much state into the application.

Do you have some ideas for an low-effort implementation of the feature?
